app/views/admin/accounts/edit.html.erb

```diff
@@ -0,0 +1,14 @@
+<div class="container">
+  <h2>Edit</h2>
+  <%= form_for([:admin, @account], html: { method: :put}) do |f| %>
```

Comment:
@gfedorenko Usually it is bad style to put brackets when you pass block to method.